### PR TITLE
Add instructions to Terms of Use screen title

### PR DESCRIPTION
**Pull request by itayplav**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203469728346481

### CODE REVIEW

File: src/common/locales/strings/enUS.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
   "recovery_instructions_complete": "To complete account recovery setup you MUST save the recovery token. This is required to recover your account in addition to your username and recovery answers. \n\n Please enter your email to be sent the recovery token.",
   "recovery_error": "The answers you provided are incorrect.",
   "read_understod_1": "I have read, understood, and agree to the ",
-  "read_understod_2": "Terms of Use",
+  "read_understod_2": "Terms of Use  (Tap circle to agree)",
```
Comment (samholmes) [inline src/common/locales/strings/enUS.json:92]: Maybe `"Terms of Use (Tap each circle to agree)"`?

**Review by samholmes — APPROVED** [review]
Approved with suggestion.

**Comment by itayplav** [comment]
> Maybe `"Terms of Use (Tap each circle to agree)"`? 

Yep I prefer this.

**Pull request merged**